src/coreclr/vm/jitinterface.cpp

```diff
@@ -11023,7 +11039,7 @@ void CEEJitInfo::reportRichMappings(
 
     JIT_TO_EE_TRANSITION();
 
-    if (m_jitManager->IsStoringRichDebugInfo())
+    if (((EEJitManager*)m_jitManager)->IsStoringRichDebugInfo())
```

Comment:
`IsStoringRichDebugInfo` lives on `EECodeGenManager` . Is this cast correct?

Reply:
You are right, this is a remainder of an intermediate change that I have later reverted.